cloud-sql/mysql/mysql/Dockerfile

```diff
@@ -15,9 +15,14 @@ WORKDIR /app
 COPY package*.json ./
 
 # Install dependencies.
-# If you add a package-lock.json speed your build by switching to 'npm ci'.
-# RUN npm ci --only=production
-RUN npm install --production
+# if you need a deterministic and repeatable build create a 
+# package-lock.json file and use npm ci:
+# RUN npm ci --omit=dev
+# if you need to include development dependencies during development
+# of your application, use:
+# RUN npm install -dev
```

Comment:
This needs to be `--dev`. Running this as is causes the following: 

```
npm WARN config dev Please use --include=dev instead.
```

This will need to be repeated in all other instances in this PR

Reply:
thanks for catching